sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Authenticated.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.restsvc.resources;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import javax.ws.rs.NameBinding;
+
+/** Annotates resources that we want filtered with {@link AuthenticationFilter}. */
+@NameBinding
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Authenticated {}
```

Comment:
Maybe `StargateAuthenticated`?

Reply:
Yeah I wasn't sure about the name, it doesn't actually perform authentication, just check that the token is present. I might go with `@CreateGrpcStub`, which is more descriptive.

Reply:
Sounds good to me, more specific to actual functioning.